enhancements/console/guided-tours.md

```diff
@@ -0,0 +1,144 @@
+---
+title: guided-tours
+authors:
+  - "@jhadvig"
+reviewers:
+  - "@spadgett"
+  - "@alimobrem"
+  - "@pweil"
+approvers:
+  - "@spadgett"
+creation-date: 2020-06-02
+last-updated: 2020-06-18
+status: provisional
+see-also:
+  - "https://issues.redhat.com/browse/CONSOLE-2255"
+  - "https://issues.redhat.com/browse/CONSOLE-2232"
+  - "https://issues.redhat.com/browse/SRVLS-262"
+---
+
+# Guided Tours
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions [optional]
+
+## Summary
+
+OpenShift's Serverless team has proposed an idea to create a "Guided Tours"
+mechanism which introduces users to various ways of interacting with serverless
+in the Console. Guided Tours should be a mechanism we can use to vastly improve
+our customer's initial user experience on a empty cluster or with all various 
+workflows:
+
+The goal of this proposal is to define a lightweight mechanism for OpenShift's
+Console component, to guide users thought various workflows, and help them
+understand the steps neccesary to get the desired outcome:
+
+* Install operator
+* Deployment of showcase application
+* Cluster settings
+* Networking
+* ...
+
+In order to achieve the desired outcome user needs to create a set of resources,
+that are dependent on each other.
+
+For Guided Tours we need a mechanism for their creation and publishment.
+
+## Motivation
+
+Help users with their understanding the principles of their workflows by
+guiding them though the necessary steps.
+
+### Goals
+
+1. Provide a mechanism to display user guides for various workflows.
+2. Make users understand what are the steps needed to achieve their goals.
+3. Provide API for writing Guided Tours.
+4. Provide new CRD format for writing the Guided Tours.
+5. Have a storage for the Guided Tours.
```

Comment:
What do you mean by "storage" here?

Reply:
Place where the openshift own Quick Starts will be stored(ideally a repository in openshift org)

Reply:
I'd call this a repository for quick starts instead of storage.

It's worth mentioning this is only needed for out-of-the-box quick starts like those describing how to install an operator. Once the operator is installed, it can add its own quick starts.